api/docs/feature-development.md

```diff
@@ -0,0 +1,238 @@
+# Feature Development in Rails
+
+## Creating a Model
+
+1.  Take a look at the erd first by running `rails erd`.
+2.  Once you think about the data model you need to create, make a test in `/specs/models`.
+
+```rb
+# frozen_string_literal: true
+
+ require 'rails_helper'
+
+ RSpec.describe NewModel, type: :model do
+ end
+
+```
+
+then, run `rails spec` and see it fail, you only should ever write just enough code to make your expectation of what should happen pass.
+
+Obviously, we can go and create the model now, because our intent is to have the model exist.
```

Comment:
I'd remove "obviously" I find the word gets a mixed bag of reactions. I feel I use this more than I should and totally understand the word choice